corehq/apps/app_manager/templates/app_manager/releases.html

```diff
@@ -376,7 +376,10 @@ <h6 data-bind="if: !built_with.signed()">{% trans "Unsigned Jar" %}</h6>
                                                 <div data-bind="ifnot: COMMCAREHQ.app_manager.versionGE(build_spec.version(), '2.13.0')">
                                                     Upgrade to CommCare 2.13 to use this feature
                                                 </div>
-                                                <div data-bind="if: COMMCAREHQ.app_manager.versionGE(build_spec.version(), '2.13.0')">
+                                                <div data-bind="ifnot: allow_media_install">
```

Comment:
It actually doesn't matter in this case. The thing you pass in here can either be an observable (in which case it's value gets accessed) or a value. Below, `()` is necessary because it's part of a larger expression